ChangeLog

```diff
@@ -1,3 +1,9 @@
+* 0.11.0
```

Comment:
We already have 11 version, this cheange log is for 0.12.0

also you need to bump version to 0.12.0a1